AntennaTracker/Parameters.cpp

```diff
@@ -250,8 +250,8 @@ const AP_Param::Info Tracker::var_info[] = {
     GOBJECT(compass,                "COMPASS_",     Compass),
 
     // @Group: SCHED_
-    // @Path: ../libraries/AP_Scheduler/AP_Scheduler.cpp
-    GOBJECT(scheduler, "SCHED_", AP_Scheduler),
+    // @Path: ../libraries/AP_Scheduler/AP_Scheduler.tpp
```

Comment:
Because it is standard for template implementation files. *.cpp can be compiled under certain conditions. But templates libraries are usually header only, if not instantiated.